Core/Module.php

```diff
@@ -24,14 +24,18 @@
  * @copyright (C) OXID eSales AG 2003-2017
  */
 
-use \OxidEsales\Eshop\Core\Module\Module;
+namespace Oxps\ModuleGenerator\Core;
+
+use OxidEsales\Eshop\Application\Model\Content;
+use OxidEsales\Eshop\Core\Model\MultiLanguageModel;
+use \OxidEsales\Eshop\Core\Module\Module as BaseModel;
```

Comment:
Perhaps better naming would be "BaseModule"

Reply:
Change naming from "BaseModel" => "BaseModule"